graylog2-server/src/main/java/org/graylog2/shared/buffers/processors/DecodingProcessor.java

```diff
@@ -143,13 +146,7 @@ private void processMessage(final MessageEvent event) throws ExecutionException
                 message = codec.decode(raw);
             }
         } catch (RuntimeException e) {
-            String remote = "unknown source";
-            final ResolvableInetSocketAddress remoteAddress = raw.getRemoteAddress();
-            if (remoteAddress != null) {
-                remote = remoteAddress.getInetSocketAddress().toString();
-            }
-            LOG.error("Unable to decode raw message {} (journal offset {}) encoded as {} received from {}.",
-                      raw.getId(), raw.getJournalOffset(), raw.getCodecName(), remote);
+            LOG.error("Unable to decode raw message on input <{}>.", raw, inputIdOnCurrentNode);
```

Comment:
Also the remote address is useful to print out, imho, or was the intention to use `RawMessage#toString` here?